build/release/verify.js

```diff
@@ -33,24 +35,34 @@ async function verifyRelease( { version } = {} ) {
 	console.log( `Verifying jQuery ${ version }...` );
 
 	let verified = true;
+	const matchingFiles = [];
+	const mismatchingFiles = [];
 
-	// Only check stable versions against the CDN
-	if ( rstable.test( version ) ) {
-		await Promise.all(
-			release.files.map( async( file ) => {
-				const cdnContents = await fetch( new URL( file.name, CDN_URL ) ).then(
-					( res ) => res.text()
-				);
-				if ( cdnContents !== file.contents ) {
-					console.log( `${ file.name } is different from the CDN:` );
-					diffFiles( file.contents, cdnContents );
+	// Check all files against the CDN
+	await Promise.all(
+		release.files
+			.filter( ( file ) => !excludeFromCDN.some( ( re ) => re.test( file.name ) ) )
```

Comment:
`!exclude...` is a double negative. I feel this would be a bit easier to read:
```suggestion
			.filter( ( file ) => excludeFromCDN.every( ( re ) => !re.test( file.name ) ) )
```